### ISSUE TITLE
Blank page on dietpi bookworm.

### ISSUE THREAD

**Issue by Hayo-Tee** [kd2org/karadav#43]
I'm trying to host karodav on my nanopi neo2, dietpi bookworm. But I get blank page. I need only local server, for backup my files by time.

I use nginx with php 8.2.

my config:

```
# /etc/nginx/sites-available/default
server {
    listen 80;
    listen [::]:80;

    server_name "$hostname";

    root /home/karadav/www;

    index index.php;

    # Serve files
    location / {
        try_files $uri $uri/ /_router.php?$query_string;
    }

    location ~* \.php$ {
        fastcgi_pass unix:/run/php/php-fpm.sock;
        include fastcgi_params;
        fastcgi_index index.php;
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
    }
}
```

**Comment by Hayo-Tee** [comment]
But first, I get:
`2023/10/03 21:59:05 [crit] 13199#13199: *37 connect() to unix:/run/php-fpm/www.sock failed (2: No such file or directory) while connecting to upstream, client: 192.168.0.103, server: dietpi, request: "GET /favicon.ico HTTP/1.1", upstream: "fastcgi://unix:/run/php-fpm/www.sock:", host: "192.168.0.20", referrer: "http://192.168.0.20/"`

then I edit "fastcgi_pass" to unix:/run/php/php-fpm.sock;, then I get no errorlog, but blank site on my host.
P.S. I'm not using reverse proxy, or other tools.

**Comment by Hayo-Tee** [comment]
Now I try with apache2, server is up. But I get Invalid url when I click manage files button. Also webdav client is can't connect, same reason

**Comment by bohwaz** [comment]
Should be fixed with 0.4.1 :)
